### ISSUE TITLE
[Feature Request]: Adding Contact Us Form 

### ISSUE THREAD

**Issue by Saipradyumnagoud** [singodiyashubham87/Draw-it-out#400]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

If we press contact us we are not able to go to contact form

### Use Case

when the user want to contact us this can be help full.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [ ] I'm a SSOC'24 contributor
- [X] I want to work on this issue

**Comment by Saipradyumnagoud** [comment]
@singodiyashubham87  can you Please Assign me this issue

**Comment by Saipradyumnagoud** [comment]
@singodiyashubham87 please assign me

**Comment by singodiyashubham87** [comment]
@Saipradyumnagoud Issue #226 is already related to that, it's a duplicate issue.

**Comment by Saipradyumnagoud** [comment]
> @Saipradyumnagoud Issue #226 is already related to that, it's a duplicate issue.
>  

I have added new page that is about us.
The issue you mentioned is different

**Comment by singodiyashubham87** [comment]
@Saipradyumnagoud As I mentioned in that issue also that we don't want a separate page for about us or any other article.

**Comment by Saipradyumnagoud** [comment]
> at we don't want a separate page for about us or any other article.

Yes ok

**Comment by singodiyashubham87** [comment]
@Saipradyumnagoud Someone is already working to make popups to display when clicked on those links, once that issue is resolved then there will be modals which will be shown whenever the user click on the links in the footer, & the issue #226 deals with the same.

**Comment by Saipradyumnagoud** [comment]
> @Saipradyumnagoud Someone is already working to make popups to display when clicked on those links, once that issue is resolved then there will be modals which will be shown whenever the user click on the links in the footer, & the issue #226 deals with the same.

yes okk let them work on

**Comment by singodiyashubham87** [comment]
@Saipradyumnagoud You can comment here again on 28th, if until then the issue is not resolved then I'll assign that issue to you :)

**Comment by Saipradyumnagoud** [comment]
> @Saipradyumnagoud You can comment here again on 28th, if until then the issue is not resolved then I'll assign that issue to you :)

Yeah sure i will

**Comment by singodiyashubham87** [comment]
@Saipradyumnagoud Yes, you can comment on the issue here: https://github.com/singodiyashubham87/Draw-it-out/issues/226
